app/javascript/debounce.js

```diff
@@ -0,0 +1,15 @@
+const debounce = (func, wait) => {
+  let timerId
+  return function (...args) {
+    if (timerId) {
+      clearTimeout(timerId)
+    }
+    timerId = setTimeout(() => {
+      func.apply(this, args)
+    }, wait)
+  }
+}
+
+export default {
+  debounce
+}
```

Comment:
関数が一つであれば下記の方がいいような気がしました。（ファイル名的にも他の関数が入るように見えないので）

```suggestion
export default debounce
```

Reply:
おっしゃる通りでした！
修正いたします。